aptos-move/move-examples/fungible_asset/sources/coin_example.move

```diff
@@ -0,0 +1,136 @@
+module aptos_framework::coin_example {
```

Comment:
please always start each file with a doc block on why, how, etc

Reply:
most of the following comments make assumptions about what this is trying to do

Reply:
sry, too lazy. I will bear that in mind.

Reply:
Yeah +1 on merging this with FungibleSource